app-admin/logstash-bin/files/gentoo-logstash-plugin

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
The script misses error handling in general. Please look into fixing that.

Also, I think it'd be reasonable to turn this into an eselect module.